pytorch_blade/bazel_build.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, *args, **kwargs):
         ]
         torch_major_version, torch_minor_version = self.torch_version.split(".")[:2]
         self.extra_opts = [
-            "--copt=-DPYTORCH_VERSION_STRING={}".format(self.torch_version),
+            "--copt=-DPYTORCH_VERSION_STRING=\\\"{}\\\"".format(self.torch_version),
```

Comment:
why need `\\\`

Reply:
We had to use `\\\` to quote strings. We had run out of the problem before since we had never use it.